core/arch/arm/include/kernel/thread.h

```diff
@@ -279,6 +280,8 @@ struct thread_handlers {
 void thread_init_primary(const struct thread_handlers *handlers);
 void thread_init_per_cpu(void);
 
+struct thread_core_local *thread_get_core_local(void);
```

Comment:
This change should go into the "core: arm: kernel: make thread_core_local public" commit